tests/Phinx/Db/TableTest.php

```diff
@@ -136,6 +141,72 @@ public function testAddTimestamps(AdapterInterface $adapter, $createdAtColumnNam
         $this->assertNull($columns[1]->getDefault());
     }
 
+    /**
+     * @dataProvider provideAdapters
+     *
+     * @param AdapterInterface $adapter
+     */
+    public function testAddTimestampsNoUpdated(AdapterInterface $adapter)
+    {
+        $table = new \Phinx\Db\Table('ntable', [], $adapter);
+        $table->addTimestamps(null, false);
+        $actions = $this->getPendingActions($table);
+
+        $columns = [];
+
+        foreach ($actions as $action) {
+            $columns[] = $action->getColumn();
+        }
+
+        $this->assertEquals(1, count($columns));
+
+        $this->assertEquals('created_at', $columns[0]->getName());
```

Comment:
We should use assertSame() for scalar values where possible.

Reply:
changed.